components/ai_chat/core/browser/tools/tool_utils.h

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_AI_CHAT_CORE_BROWSER_TOOLS_TOOL_UTILS_H_
+#define BRAVE_COMPONENTS_AI_CHAT_CORE_BROWSER_TOOLS_TOOL_UTILS_H_
+
+#include <string>
+#include <vector>
+
+#include "brave/components/ai_chat/core/common/mojom/ai_chat.mojom-forward.h"
+#include "url/gurl.h"
+
+namespace ai_chat {
+
+std::vector<mojom::ContentBlockPtr> CreateContentBlocksForText(
+    const std::string& text);
+
+std::vector<mojom::ContentBlockPtr> CreateContentBlocksForImage(
+    const GURL& image_url);
```

Comment:
One of them is only used on test and another is unused. Do we really need to pull these out as separate utils especially since they are trivial?

Reply:
Well I had them in my stash because every tool will use them to provide the tool results instead of manually making a vector and pushing to it every time, so they are useful. And they will get used more... tomorrow... what do you think?